extra/nrnivmodl_core_makefile.in

```diff
@@ -187,9 +187,9 @@ $(SPECIAL_EXE): coremech_lib_target
 	  -Wl,-rpath,'$(LIB_RPATH)' -Wl,-rpath,$(CORENRN_LIB_DIR)
 
 coremech_lib_target: $(corenrnmech_lib_target)
-	rm -rf $(OUTPUT_DIR)/.libs; \
+	rm -rf $(OUTPUT_DIR)/.libs/libcorenrnmech$(LIB_SUFFIX); \
```

Comment:
Nitpick.. To be consistent with `COREMECH_LIB_PATH` maybe this can be:
```suggestion
	rm -rf $(OUTPUT_DIR)/.libs/lib$(COREMECH_LIB_NAME)$(LIB_SUFFIX); \
```